### PR TITLE
17 cancel registration

### PR DESCRIPTION
**Pull request by domcermak**
#17

### CODE REVIEW

**Review by BelinaJaroslav — APPROVED** [review]
lgtm, fixed typo in TUI

File: src/app/formatters/HelpTextFormatter.java [hunk 1]
```diff
@@ -6,7 +6,7 @@ public String format() {
       return "Vaccination register options:\n"
             + String.format("%-110s %s\n", "list (patients | doctors | vaccines | hospitals | hospitals_doctors | registered_vaccinations) [id]", "Lists records in selected entity")
             + String.format("%-110s %s\n", "delete_patient (patient_id)", "Deletes patient and it's registrations from the registry")
-            + String.format("%-110s %s\n", "cancel_registration (patient_id)", "Deletes registered vaccination from registry")
```
Comment (domcermak) [inline src/app/formatters/HelpTextFormatter.java]: This was correct. This serves to delete single registered vaccination, not all registered vaccinations of a patient

Comment (domcermak) [reply src/app/formatters/HelpTextFormatter.java]: So perhaps the command name is wrong, but the command description should be ok.

**Comment by BelinaJaroslav** [comment]
Screenshot from 2021-01-07 17-03-40
When I try to use the program on my database, I have to put in the ID of the registration as an argument for the function. Thats why I think specyfing the argument as registration_ID instead of patient_ID. 
cancel_registration (patient id) looks to me, as an user like it would delete all registration for given id of patient

**Comment by domcermak** [comment]
Yeah, right. My mistake. I will probaly change it to `registered_vaccination_id`

**Review by BelinaJaroslav — APPROVED** [review]
lgtm now

**Pull request merged**